src/wp-includes/blocks/template-part.php

```diff
@@ -114,7 +114,11 @@ function render_block_core_template_part( $attributes ) {
 	$is_debug = WP_DEBUG && WP_DEBUG_DISPLAY;
 
 	if ( is_null( $content ) ) {
-		if ( $is_debug && isset( $attributes['slug'] ) ) {
+		if ( $is_debug ) {
+			if ( ! isset( $attributes['slug'] ) ) {
+				// If there is no slug this is a placeholder and we dont want to return any message.
+				return '';
+			}
```

Comment:
@Rajinsharwar this file is maintained within Gutenberg packages. Please create a PR in the Gutenberg repo. The change can be made in `packages/block-library/src/template-part/index.php` file.

https://github.com/WordPress/gutenberg/blob/ee30a87abd733d00338730eed831cfc02d27edf0/packages/block-library/src/template-part/index.php#L117

Reply:
Will do @hellofromtonya. Should I create a new issue and assign the PR there? Not sure if there's an existing one.

Reply:
@Rajinsharwar here's the issue for it in Gutenberg https://github.com/WordPress/gutenberg/issues/62559

Reply:
PR created @hellofromtonya: https://github.com/WordPress/gutenberg/pull/62568